### DISCUSSION TITLE
Can a similar system be made to prerender Photoshop/Premiere Pro

### DISCUSSION THREAD

**Discussion by blueprintparadise** [inlife/nexrender#639] [General]
I was wondering if it is possible to make something similar to this project for other Adobe products.
If yes , how would you recommend one should go about it.

**Comment by inlife** [comment]
Hey @blueprintparadise 

How would you use such a system for Photoshop or Premiere Pro, just out of curiosity? :)

**Reply by blueprintparadise** [reply]
Not exactly a rendering program, but an interface that helps automate tasks without extensive Extendscript coding

**Reply by inlife** [reply]
I think in theory something like this can be done, sure. That however most likely would be a different kind of project at that point.
